app/views/cash_account_no_transactions_with_balance.scala.html

```diff
@@ -44,9 +44,4 @@ <h1 class="govuk-heading-xl">@messages("cf.cash-account.detail.heading")</h1>
             @messages("cf.cash-account.detail.link.post")</p>
     </div>
 
-    <h2 id="missing-documents-guidance-heading" class="govuk-heading-m govuk-!-margin-top-9">@messages("cf.cash-account.transactions.request.link.heading")</h2>
-    <p class="govuk-body govuk-!-padding-bottom-7">
-        @messages("cf.cash-account.transactions.request.link.pre") <a class="govuk-link" href="@controllers.routes.CashTransactionsRequestPageController.onPageLoad">
-    @messages("cf.cash-account.transactions.request.link")</a>
-    </p>
```

Comment:
This one should not be removed as if they have a balance it means they are likely to have previous transactions